docs/src/development/cli/_index.md

```diff
@@ -14,52 +14,24 @@ Its source code is hosted on [GitHub](https://github.com/platformsh/platformsh-c
 
 ## Before you begin
 
-You need to have:
-
-* PHP 5.5.9+
-* Git
-* A Bash-like shell
-
-  On Windows, the best way to get Bash is through [Windows Subsystem for Linux](https://msdn.microsoft.com/en-gb/commandline/wsl/about).
-  You can also use another Bash-compatible shell such as [Git Bash](https://gitforwindows.org/),
-  which you might have installed with git.
-
-For full requirements, see the [requirements on GitHub](https://github.com/platformsh/platformsh-cli#user-content-requirements).
+{{% cli-installation-requirements %}}
 
 ## 1. Install
 
-Install the CLI using this command:
-
-```bash
-curl -fsS https://platform.sh/cli/installer | php
-```
-
-If you run into issues or want to install manually, see the [installation instructions on GitHub](https://github.com/platformsh/platformsh-cli#user-content-installation).
+{{% cli-installation %}}
 
 ## 2. Authenticate
 
-To see and manage projects, you need to be authenticated with Platform.sh
-Run this command:
-
-```bash
-platform
-```
-
-You are asked to log in via a browser.
-This process creates SSH certificates on your computer for secure connections.
-(See more about [SSH connections](/development/ssh/_index.md).)
```

Comment:
I think the bit about SSH connections makes sense when you're explaining what the CLI is. So here. But not in a getting started tutorial where the goal is about something else and you don't care about the details of the CLI.

I wouldn't do 2 because I'd want the getting started tutorials to be self-contained as much as possible.

Reply:
Well, it's not completely necessary to explicitly use `login` as this seems to say. I'd suggest something more like

````
To see and manage projects, authenticate by running the following command:

```bash
platform login
```

This process opens a browser tab for you to log in.
It also creates certificates on your computer for [SSH connections](/development/ssh/_index.md).
````

Given that SSH = "secure shell", "secure SSH connections" feels like "ATM machines".

I think it's fine as just `platform` rather than `platform login`, but either way works :shrug: